source/elements/oneMKL/source/domains/rng/device_routines/device-rng-generate-single.rst

```diff
@@ -0,0 +1,68 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_generate_single:
+
+generate_single
+===============
+
+
+Entry point to obtain a single random number from a given vector engine with proper statistics of a given distribution.
+
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+API
+***
+
+
+Syntax
+------
+
+.. code-block:: cpp
+
+   template<typename Distr, typename Engine>
+   typename Distr::result_type generate_single(Distr& distr, Engine& engine)
```

Comment:
Add `;` at the end?

Reply:
+ namespace

Reply:
Removed `fenerate_single` and applied to `generate`